install/mlops/truera/install.yml

```diff
@@ -0,0 +1,16 @@
+id: truera-mlops
+name: Truera Inc
+
+title: Truera diagnositcs and monitoring solution for Trustworthy AI.
+
+description: |
+  TruEra provides best in class AI Quality diagnostics and monitoring solution. With this integration, TruEra and NewRelic users can go beyond standard accuracy, input and output drift monitoring by identifying consequential data drift, and other AI Quality diagnostics such as fairness, conceptual soundness and segment behavior. When combined with NewRelic’s comprehensive observability capabilities, MLOps teams can correlate signals throughout the ML lifecycle.
```

Comment:
This is a pretty long run on sentence, would you mind trying to break this up into shorter sentences?

Reply:
You can try: 
`TruEra provides best in class AI quality diagnostics and monitoring solution.`